test/tstDetailsRay.cpp

```diff
@@ -15,27 +15,33 @@
 
 #define BOOST_TEST_MODULE DetailsRay
 
+BOOST_AUTO_TEST_CASE(ieee)
+{
+  float pinf = 1.0 / 0.0;
+  float ninf = -1.0 / 0.0;
+  float zerotimespinf = 0 * pinf;
+  float zerotimesninf = 0 * ninf;
+  BOOST_TEST(!(zerotimespinf == zerotimespinf));
+  BOOST_TEST(!(zerotimesninf == zerotimesninf));
+  BOOST_TEST(!(ninf < zerotimespinf));
+  BOOST_TEST(!(pinf < zerotimespinf));
+  BOOST_TEST(!(ninf > zerotimesninf));
+  BOOST_TEST(!(pinf > zerotimesninf));
+  BOOST_TEST(!(1.0 < zerotimespinf));
+  BOOST_TEST(!(1.0 > zerotimespinf));
+  BOOST_TEST(!(-1.0 > zerotimesninf));
+  BOOST_TEST(!(-1.0 > zerotimesninf));
+}
+
 BOOST_AUTO_TEST_CASE(intersects_box)
 {
   using ArborX::Box;
   using ArborX::Point;
   using ArborX::Ray;
-  constexpr auto &intersects = ArborX::Details::RayTracing::intersects;
 
-  float pinf = 1.0 / 0.0;
-  float ninf = -1.0 / 0.0;
-  float zero_pinf = 0 * pinf;
-  float zero_ninf = 0 * ninf;
-  BOOST_TEST(!(zero_pinf == zero_pinf));
-  BOOST_TEST(!(zero_ninf == zero_ninf));
-  BOOST_TEST(!(ninf < zero_pinf));
-  BOOST_TEST(!(pinf < zero_pinf));
-  BOOST_TEST(!(ninf > zero_ninf));
-  BOOST_TEST(!(pinf > zero_ninf));
-  BOOST_TEST(!(1.0 < zero_pinf));
-  BOOST_TEST(!(1.0 > zero_pinf));
-  BOOST_TEST(!(-1.0 > zero_ninf));
-  BOOST_TEST(!(-1.0 > zero_ninf));
+  auto intersects = [](Ray r, Box b) {
+    return ArborX::Details::intersects(r, b);
+  };
```

Comment:
```C++
using ArborX::Details::intersects;

Reply:
I can no longer do that because of function overloading.
There are other intersects() functions under the namespace ArborX::Details

Reply:
Should not matter

Reply:
You are right. It doesn't matter.